frontends/main/next.config.js

```diff
@@ -13,6 +29,7 @@ const nextConfig = {
       new webpack.IgnorePlugin({
         resourceRegExp: /mockAxios\.ts/,
       }),
+      new webpack.EnvironmentPlugin(processFeatureFlags()),
```

Comment:
I think we should avoid adding to the webpack config unless we really need to, and NextJS has an `env` key for this... Could  `processFeatureFlags ` return the bootstrapFeatureFlags, and then do:
```js
const nextConfig = {
  webpack: { ... } // leave as is... I'm pretty sure we can get rid of it entirely, but no need to futz with it here.
  env: {
    FEATURE_FLAGS: JSON.stringify(bootstrapFeatureFlags)
  }
  ...
}
```